crates/bitcoin-da/tests/bitcoin_da_tests.rs

```diff
@@ -386,47 +439,149 @@ mod tests {
         }
     }
 
+
     #[test]
-    fn test_read_height() {
+    fn test_read_data_by_block_height() {
```

Comment:
great change for regtest !
Will not work for signet unfortunately, that's why we preferred to hardcode the block height.

Reply:
I can have a single test where we can have both the check for regtest and the hardcode for blockheight and keep the easy toggle at the command line level. Would this be something that would make sense for you guys to have?

Reply:
I'm good with that !